docs/src/sections/NavbarSection.js

```diff
@@ -41,12 +41,14 @@ export default function NavbarSection() {
 
       <h3><Anchor id="navbars-mobile-friendly">Responsive Navbars</Anchor></h3>
       <p>
-        To have a mobile friendly Navbar, Add a <code>Navbar.Toggle</code> to your Header and wrap your
-        Navs in a <code>Navbar.Collapse</code> component. The <code>Navbar</code> will automatically wire
-        the toggle and collapse together!
+        To have a mobile-friendly Navbar, add a <code>Navbar.Toggle</code> to your Header and wrap your
+        Navs in a <code>Navbar.Collapse</code> component. The <code>Navbar</code> will automatically combine
+        the toggle and collapse.
       </p>
       <p>
-        Set the <code>defaultExpanded</code> prop to make the Navbar start expanded. Set <code>collapseOnSelect</code> to make the Navbar collapse automatically when the user selects an item. You can also finely control the collapsing behavior by using the <code>expanded</code> and <code>onToggle</code> props.
+        Set the <code>defaultExpanded</code> prop to make the Navbar start expanded. Set <code>collapseOnSelect</code>
```

Comment:
Usually editors can be set to wrap lines. I've come around to the idea that it's better to not manually wrap lines – otherwise for line length limits, you end up potentially having to touch a lot of lines of docs source just to add or remove a new word or something.